Dangerfile

```diff
@@ -1,46 +1,31 @@
 # frozen_string_literal: true
 
-def wip_feature?
-  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
-  has_wip_title = github.pr_title.include?('WIP')
+github.dismiss_out_of_range_messages
 
-  has_wip_label || has_wip_title
-end
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
```

Comment:
Maybe it's possible to do what in Swift would be an `extension` and I think Ruby calls [monkey patch](https://en.wikipedia.org/wiki/Monkey_patch) to add a method to the RuboCop plugin that would go like:

```ruby
rubocop.lint_all_files(force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
```

This is of course a Dangermattic-level idea, which is out of scope for this PR.

Reply:
Good suggestion! I've added an item to the backlog [here](https://github.com/orgs/Automattic/projects/775?pane=issue&itemId=53114939).